Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -555,8 +555,8 @@ private bool SameOrLesserPluginVersionExists(string metadataPath)
             var newMetadata = JsonSerializer.Deserialize<PluginMetadata>(File.ReadAllText(metadataPath));
             return Context.API.GetAllPlugins()
                                 .Any(x => x.Metadata.ID == newMetadata.ID 
-                                    && (x.Metadata.Version == newMetadata.Version) 
-                                        || newMetadata.Version.CompareTo(x.Metadata.Version) < 0);
+                                    && (x.Metadata.Version == newMetadata.Version 
+                                        || newMetadata.Version.CompareTo(x.Metadata.Version) < 0));
```

Comment:
good pick up @Garulf , i remmember i was thinking of needing to put them into brackets, not sure why i then forgot lol.

Reply:
No worries, glad we caught it.

Reply:
A simply <=0 will be fine